autodetect-dependabot.jl

```diff
@@ -0,0 +1,29 @@
+module AutodetectDependabot
+
+function get_possible_branch_names()
+    possible_branch_names = [
+        get(ENV, "GITHUB_BASE_REF", ""),
+        get(ENV, "GITHUB_HEAD_REF", ""),
+        get(ENV, "GITHUB_REF", ""),
+    ]
+    return possible_branch_names
+end
+
+function chop_refs_head(branch_name::AbstractString)
+    if startswith(branch_name, "refs/heads/")
+        return replace(branch_name, "refs/heads/" => "")
```

Comment:
For the (very unlikely) case that someone names their branch `refs/heads`, let's only replace the first occurence and only if it's at the start of the string:

```suggestion
        return replace(branch_name, r"^(refs\/heads\/)" => "")
```